tools/goctl/model/sql/template/tpl/model-new.tpl

```diff
@@ -5,9 +5,3 @@ func new{{.upperStartCamelObject}}Model(conn sqlx.SqlConn{{if .withCache}}, c ca
 	}
 }
 
-func (m *default{{.upperStartCamelObject}}Model) withSession(session sqlx.Session) *default{{.upperStartCamelObject}}Model {
```

Comment:
Considering compatibility, I don't think it should be removed here, and then create a new exported method and call this method in customeModel.